Templates/integration_template.ps1

```diff
@@ -0,0 +1,74 @@
+<#
+.Synopsis
+   Template for creating DSC Resource Integration Tests
+.DESCRIPTION
+   To Use:
+     1. Copy to \Tests\Integration\ folder and rename MSFT_x<ResourceName>.Integration.tests.ps1
+     2. Customize TODO sections.
+     3. Create test DSC Configurtion file MSFT_x<ResourceName>.config.ps1 from integration_config_template.ps1 file.
+
+.NOTES
+   Code in HEADER, FOOTER and DEFAULT TEST regions are standard and may be moved into
+   DSCResource.Tools in Future and therefore should not be altered if possible.
+#>
+
+# TODO: Customize these paramters...
+$DSCModuleName      = 'x<ModuleName>' # Example xNetworking
+$DSCResourceName    = 'MSFT_x<ResourceName>' # Example MSFT_xFirewall
+# /TODO
+
+#region HEADER
+Import-Module .\DSCResource.Tests\TestHelper.psm1 -Force
```

Comment:
I don't love this to be honest, wouldn't this mean we have to have a DSCResource.Tests in the unit\integration folder in order to properly import this if we took the template whole sale?

Thinking around this I see a few options.

1) Remove the import-module and put in a comment where to download/clone DSCResource.Tests - this puts the onus on the test runner to setup the system before the tests run successfully. It also means we don't automatically setup the system for them which may not be cool to some people and has been something we've avoided in the past.

2) We try to automatically setup the system to run the tests, this means we'd have to clone/install DscResource.Tests ourselves, this would add more overhead to our code and would mean checking/installing during every integration/unit test.  Also means our git repos get dirtied up with a extra cruft if we put it directly in the folder so we'd probably want to remove the file when we're done.